common/changes/@visactor/vrender/fix-FadeInPlus_2023-07-03-07-56.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@visactor/vrender",
+      "comment": "fix the bug in FadeInPlus",
```

Comment:
It's better to use `fix: fix the bug in FadeInPlus` , just the same as commit message 
then the message can be parsed in release log